client/my-sites/plans/jetpack-plans/product-card/index.tsx

```diff
@@ -236,8 +250,25 @@ const ProductCard: React.FC< ProductCardProps > = ( {
 			scrollCardIntoView={ scrollCardIntoView }
 			collapseFeaturesOnMobile={ collapseFeaturesOnMobile }
 			pricesAreFetching={ pricesAreFetching }
+			isPricingPageTreatment202204={ isPricingPageTreatment202204 }
+			isPricingPageTest202204AssignmentLoading={ isLoadingExperimentAssignment }
+			belowButtonText={
+				isPricingPageTreatment202204 ? translate( 'Renews at the normal rate.' ) : ''
+			}
 		/>
 	);
 };
 
-export default ProductCard;
+export default connect( ( state ) => {
+	const siteId = getSelectedSiteId( state );
+	const currentRoute = siteId
+		? getCurrentRouteParameterized( state, siteId )
+		: getCurrentRoute( state );
+
+	return {
+		isJetpackPricingPage:
```

Comment:
This seems fine to me. Alternatively, you could pass a boolean prop to `ProductGrid` in the [`SelectorPage`](https://opengrok.a8c.com/source/xref/calypso/client/my-sites/plans/jetpack-plans/selector.tsx?r=95fa44ce), and propagate it to `ProductCard`. The `SelectorPage` is used by the 3 pricing pages (Calypso, Calypso Jetpack connect, and cloud).

Reply:
Thanks @monsieur-z. I was trying to be as specific as possible here to make sure it's only shown on these pages, but maybe it's overkill. Would doing it the way you describe guarantee that it's only shown on these pages and so remove any need for this whitelist?

Reply:
Yes, the `SelectorPage` component is used by the (three) pages that display the Jetpack products listing, and only them.

Reply:
Cool, that seems like a slightly cleaner approach then. I'll make some updates.